src/app/[id]/page.module.scss

```diff
@@ -0,0 +1,9 @@
+.main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2rem;
+  min-height: 100vh;
```

Comment:
Please use dvh instead of vh. In the following video you can find out why 
https://www.youtube.com/watch?v=xl9R8aTOW_I&list=PLNYkxOF6rcIAKIQFsNbV0JDws_G_bnNo9&index=4